hr_course/tests/test_hr_course.py

```diff
@@ -17,49 +17,61 @@ def setUp(self):
             {
                 "name": "Course name",
                 "category_id": self.course_categ.id,
-                "cost": 100,
-                "authorized_by": self.employee1.id,
                 "permanence": True,
                 "permanence_time": "1 month",
+            }
+        )
+        self.course_schedule_id = self.env["hr.course.schedule"].create(
+            {
+                "name": "Convocatory",
+                "course_id": self.course_id.id,
+                "cost": 100,
```

Comment:
cost for the schedule not for course?

Reply:
The cost is per course so as not to have to duplicate the course if the price changes annually or depending on where the course is held.